app/(home)/plans/[branche]/page.tsx

```diff
@@ -29,7 +30,7 @@ export default async function Page(props: PageProps) {
   }
 
   branche.plans = await getPlans(branche);
-  if (!branche.plans.length) {
+  if (branche.plans.length) {
```

Comment:
```suggestion
  if (!branche.plans.length) {
```
É para renderizar essa tela quando não existir planos.